syntax = "proto3";

package FoodBotSim;

service FoodBotSimRequest
{
//Input NLG
//Output NLG 
	rpc GetSimResponse(Sentence) returns (Sentence){}
}

message Sentence
{
	string response = 1;
}
